test/utility/test_util.i

```diff
@@ -108,8 +111,15 @@ testutil_clean_work_dir(char *dir)
 		testutil_die(ENOMEM, "Failed to allocate memory");
 
 	snprintf(buffer, inputSize, "%s%s", RM_COMMAND, dir);
-	if ((ret = system(buffer)) != 0)
-		testutil_die(ret, "System call to remove directory failed");
+
+	existp = 0;
```

Comment:
This variable should be ```exist```, the ```p``` at the end indicates a pointer - and the local variable isn't a pointer.